modules/rosa-common-commands.adoc

```diff
@@ -33,10 +33,10 @@ $ rosa download <software>
 [id="rosa-help_{context}"]
 == help
 
-Displays general help information for the `rosa` CLI and a list of available commands. This option can also be used as an argument to display help information for a parent command, such as `version` or `create`.
+Displays general help information for the {product-title} (ROSA) CLI (`rosa`) and a list of available commands. This option can also be used as an argument to display help information for a parent command, such as `version` or `create`.
```

Comment:
This instance can just be "ROSA CLI"

Reply:
Maybe "ROSA CLI (`rosa`)" as this is the first instance and then later "ROSA CLI".